common/config.py

```diff
@@ -129,10 +129,28 @@ class Config(configfile.ConfigFileWithProfiles):
         DISK_UNIT_GB : 'GiB'
     }
 
-    DEFAULT_EXCLUDE = [ '.gvfs', '.cache/*', '.thumbnails*',
-    '.local/share/[Tt]rash*', '*.backup*', '*~', '.dropbox*', '/proc/*',
-    '/sys/*', '/dev/*', '/run/*', '/etc/mtab', '/var/cache/apt/archives/*.deb',
-    'lost+found/*', '/tmp/*', '/var/tmp/*', '/var/backups/*', '.Private' ]
+    # Used when new snapshot profile is created.
+    DEFAULT_EXCLUDE = [
+        '.gvfs',
+        '.cache/*',
+        '.thumbnails*',
+        '.local/share/[Tt]rash*',
+        '*.backup*',
+        '*~',
+        '.dropbox*',
+        '/proc/*',
+        '/sys/*',
+        '/dev/*',
+        '/run/*',
+        '/etc/mtab',
+        '/var/cache/apt/archives/*.deb',
+        'lost+found/*',
+        '/tmp/*',
+        '/var/tmp/*',
+        '/var/backups/*',
+        '.Private',
+        'swapfile',
```

Comment:
Should the exclude pattern be restricted to the root folder (= default location of the swap file)?

Reply:
I thought about that, too. But I assume that some people do locate it in secondary mounted volume (e.g. `/mnt/anotherhdd/swapfile`) or something like that.

On the other hand I thought if there are files named "swapfile" somewhere else with different purposes.

Maybe it is saver to restrict it to root folder with `/swapfile` ?